src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/DocumentTransformer.kt

```diff
@@ -17,25 +17,14 @@ class DocumentTransformer {
   fun transformToApiFiltered(groupedDocuments: GroupedDocuments, convictionId: Long?) =
     transformToApi(groupedDocuments) { it.convictionId == convictionId.toString() }
 
-  fun transformToApiUnfiltered(groupedDocuments: GroupedDocuments) =
-    transformToApi(groupedDocuments) { true }
+  fun transformToApiUnfiltered(groupedDocuments: GroupedDocuments, onlyConvictionDocuments: Boolean) =
+    transformToApi(groupedDocuments, onlyConvictionDocuments) { true }
```

Comment:
This function is called transformToApiUnfiltered, but it's now doing filtering.

I wonder if it would be cleaner to remove transformToApiFiltered and transformToApiUnfiltered and have callers call transformToApi directly?